app/wmt-etl/poll-sqs.js

```diff
@@ -28,21 +23,19 @@ module.exports = function () {
           throw new Error('ETL already running')
         }
         return deleteSqsMessage(sqsClient, queueURL, data.Messages[0].ReceiptHandle).then(function () {
-          return listObjects(s3Client, S3.BUCKET_NAME).then(function (extracts) {
-            const excelExtracts = extracts.filter((extract) => extract.Key.endsWith('.xlsx'))
-            if (bothFilesPresent(excelExtracts)) {
-              return Promise.all(excelExtracts.map((extract) => getHasBeenRead(extract.Key)))
-                .then(function ([first, second]) {
-                  if (!first && !second) {
-                    return runEtl()
-                  } else {
-                    log.info('Files have been read')
-                    return 'Files have been read'
-                  }
-                })
+          return listObjects(s3Client, S3.BUCKET_NAME, S3.FILE_TO_PROCESS).then(function (extracts) {
```

Comment:
there's only one file so we could use getS3Object and pass it into runEtl
Might be simpler? It's a bit of a refactor I appreciate.
We would still have to do the tagging too.
Just a thought, not convinced it is worthwhile